app/src/test/java/dev/msfjarvis/lobsters/data/local/SqlDelightQueriesTest.kt

```diff
@@ -0,0 +1,203 @@
+
+    // Check post count
+    val postsCount = postQueries.selectCount().executeAsOne()
+    assertEquals(1, postsCount)
+
+    // Check if post is updated
+    val postFromDb = postQueries.selectPost(post.short_id).executeAsOne()
+    assertEquals(true, postFromDb.isSaved)
+  }
+
+  @Test
+  fun selectPost() = runBlocking {
+    // Get 1 post
+    val post = createTestData(1)[0]
+
+    // Insert post into DB
+    postQueries.insertOrReplacePost(post)
+
+    val postFromDb = postQueries.selectAllPosts().executeAsOne()
+    assertEquals("test_id_1", postFromDb.short_id)
+  }
+
+  @Test
+  fun selectAllPosts() = runBlocking {
+    // Get 5 post
+    val posts = createTestData(5)
+
+    // Insert posts into DB
+    posts.forEach { postQueries.insertOrReplacePost(it) }
+
+    val postsFromDb = postQueries.selectAllPosts().executeAsList()
+
+    // Check if all posts have correct short_id
+    for (i in 1..5) {
+      assertEquals("test_id_$i", postsFromDb[i - 1].short_id)
+    }
+  }
+
+
+  @Test
+  fun savePost() = runBlocking {
+    // Insert post to DB
+    val post = createTestData(1)[0]
+    postQueries.insertOrReplacePost(post)
+
+    // Update isSaved property of post
+    postQueries.savePost(post.short_id)
+
+    // Get the post and check if isSaved is true
+    val postFromDB = postQueries.selectPost(post.short_id).executeAsOne()
+    assertEquals(true, postFromDB.isSaved)
+  }
+
+  @Test
+  fun removeSavedPost() = runBlocking {
+    val post = createTestData(1)[0]
+    postQueries.insertOrReplacePost(post)
+
+    postQueries.savePost(post.short_id)
+
+    // Get the post and check if isSaved is true
+    val postFromDB = postQueries.selectPost(post.short_id).executeAsOne()
+    assertEquals(true, postFromDB.isSaved)
+
+    postQueries.removeSavedPost(post.short_id)
+
+    // Get the post and check if isSaved is false
+    val updatedPostFromDB = postQueries.selectPost(post.short_id).executeAsOne()
+    assertEquals(false, updatedPostFromDB.isSaved)
+  }
+
+  @Test
+  fun deletePost() = runBlocking {
+    // Create 3 posts and insert them to DB
+    val posts = createTestData(3)
+    posts.forEach { postQueries.insertOrReplacePost(it) }
+
+    // Delete 2nd post
+    postQueries.deletePost("test_id_2")
+
+    val postsFromDB = postQueries.selectAllPosts().executeAsList()
+
+    // Check if size is 2, and only the correct post is deleted
+    assertEquals(2, postsFromDB.size)
+    assertEquals("test_id_1", postsFromDB[0].short_id)
+    assertEquals("test_id_3", postsFromDB[1].short_id)
+  }
+
+  @Test
+  fun deleteAllPost() = runBlocking {
+    // Create 3 posts and insert them to DB
+    val posts = createTestData(5)
+    posts.forEach { postQueries.insertOrReplacePost(it) }
+
+    // Delete 2nd post
+    postQueries.deleteAllPosts()
+
+    val postsCount = postQueries.selectCount().executeAsOne()
+
+    // Check if size is 2, and only the correct post is deleted
+    assertEquals(0, postsCount)
+  }
```

Comment:
This entire test confuses me

Reply:
Don't read the comments lol. I copied and updated the code but forgot to update the comment